jetpack-io/devbox
# Added prompt for direnv allow after .envrc

## Summary
Title says it
This is step 3 out of 4 for simplifying direnv integration
## How was it tested?
- compile
- run devbox init
- confirm it prompts for direnv integration if direnv is installed
